Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/gnucash/-/gnucash/horizontal-auto.svg)

fellen commented on po/glossary/pt_BR.po:
```diff
@@ -84,7 +84,7 @@ msgstr "Receita"
 #. "A debt, a financial obligation, but see also 'account type: Passive'"
 msgid "account type: Liability"
-msgstr "Passivo"
+msgstr "tipo de conta: Responsabilidade"
```
@wtuemura Please remove [Edit] the generic terms like `tipo de conta: ` here in the whole file [/Edit] See the [instructions in the wiki](https://wiki.gnucash.org/wiki/Translation#The_glossary_file).

wtuemura committed:
Translation update  by Wellington Terumi Uemura <wellingtonuemura@gmail.com> using Weblate

po/glossary/pt_BR.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Portuguese (Brazil))
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/pt_BR/

Translation update  by Wellington Terumi Uemura <wellingtonuemura@gmail.com> using Weblate

po/glossary/pt_BR.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Portuguese (Brazil))
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/pt_BR/

Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>

6e2f61 committed:
Translation update  by vim <dev.getport@slmail.me> using Weblate

po/pl.po: 70.7% (3906 of 5524 strings; 1076 fuzzy)
299 failing checks (5.4%)
Translation: GnuCash/Program (Polish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/pl/

Co-authored-by: vim <dev.getport@slmail.me>

SziaTomi committed:
Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/glossary/hu.po: 100.0% (216 of 216 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/hu/

Co-authored-by: Szia Tomi <sziatomi01@gmail.com>

CWehli committed:
Translation update  by Christian Wehling <christian.wehling@web.de> using Weblate

po/glossary/de.po: 100.0% (216 of 216 strings; 0 fuzzy)
5 failing checks (2.3%)
Translation: GnuCash/Glossary (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/de/

Translation update  by Christian Wehling <christian.wehling@web.de> using Weblate

po/de.po: 99.3% (5490 of 5524 strings; 29 fuzzy)
602 failing checks (10.8%)
Translation: GnuCash/Program (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/de/

Co-authored-by: Christian Wehling <christian.wehling@web.de>

fellen commented on po/pl.po:
```diff
@@ -5361,7 +5361,7 @@ msgstr "Opcje GnuCash"
 #: gnucash/report/reports/standard/invoice.scm:849
 msgid "Printable Invoice"
-msgstr "Drukowalna faktura"
+msgstr "Faktura (do druku)"
```
@6e2f61 IMHO it is a bad idea to use brackets in the glossary if they do not exist in the english messaage.

fellen commented:
Thanks!
